Gemfile

```diff
@@ -86,8 +86,8 @@ group :heroku do
   gem 'puma'
 end
 
-gem 'responds_to_parent', git: 'https://github.com/CloCkWeRX/responds_to_parent.git', branch: 'patch-2' # Temporarily pointed at git until https://github.com/zendesk/responds_to_parent/pull/7 is released
-gem 'acts_as_commentable', git: 'https://github.com/fatfreecrm/acts_as_commentable.git', branch: 'rails-61' # Our fork
+gem 'responds_to_parent'
```

Comment:
Ditto this one. Zendesk appear to be a bit useless at merging.

https://github.com/zendesk/responds_to_parent/pull/7